app/views/admin/users/index.html.erb

```diff
@@ -34,7 +34,7 @@
                 <%= user_image(user, variant: :large) %>
               </div>
               <div>
-                <div><%= user.soft_deleted? ? user.login : link_to(user.login, user_home_path(user.login)) %></div>
+                <div><%= user.soft_deleted? ? user.login : link_to(user.login + " (#{user.display_name})", user_home_path(user.login)) %></div>
```

Comment:
Our admin is not the prettiest situation in the moment, but I don't think there's enough horizontal space to add display_name here. We could just use `user.display_name` instead of `login` — this is the pattern used across the site: https://github.com/sudara/alonetone/blob/master/app/models/user.rb#L80

<img width="443" alt="image" src="https://user-images.githubusercontent.com/472/66574347-5e292800-eb74-11e9-8b95-148f113d512a.png">